helm/ffc-pay-statement-receiver/values.yaml

```diff
@@ -35,6 +35,7 @@ container:
   redisHostname: hosty-mchost-face
   redisPort: 9999
   redisPassword: redis
+  endpointEnabled: true
```

Comment:
Do we havva AppConfig value for this set in pipeline?

Reply:
We do for PRE and PROD where it's set to `false` currently until we're ready.  For lower environments with no real data will leave open to support integration with CRM testing.